src/etlng/impl/TaskManager.cpp

```diff
@@ -107,13 +115,24 @@ TaskManager::spawnExtractor(util::async::AnyStrand& strand, PriorityQueue& queue
 }
 
 util::async::AnyOperation<void>
-TaskManager::spawnLoader(PriorityQueue& queue)
+TaskManager::spawnLoader(TaskQueue& queue)
 {
     return ctx_.execute([this, &queue](auto stopRequested) {
         while (not stopRequested) {
             // TODO (https://github.com/XRPLF/clio/issues/66): does not tell the loader whether it's out of order or not
-            if (auto data = queue.dequeue(); data.has_value())
-                loader_.get().load(*data);
+            if (auto data = queue.dequeue(); data.has_value()) {
+                auto nanos = util::timed<std::chrono::nanoseconds>([this, data = *data] { loader_.get().load(data); });
+                auto const seconds = nanos / 1000000000.0;
```

Comment:
Would be good to move it into a named constant and use scientific notation `1.0e9`.